### ISSUE TITLE
Add Confirmation Dialog

### ISSUE THREAD

**Issue by bravozulu1** [TroyaltyCorp/TLC_Chrome_Plugin#2]
Add confirmation for existing user entry and warn the user with a pop-up dialog asking if they want to overwrite existing password.   If you manually enter the password, submit, login failed because of mistyped password, the bad password will get saved.  Without the user knowing.

**Comment by AntonFisher1** [comment]
@bravozulu1 
Could you please clarify what kind of confirmation/warning do you mean? What design, content and buttons should it contain, and when exactly should it be shown?

As I understood, you want to show one new kind of pop-up only in case if user tries to log in with some existing username from DB, but with different password. Am I right? If yes - which UI design, content and buttons do you expect on this pop-up?

P.S. As I can see (at least with Password Vault app v. 2.36.1010 for Windows 7, not sure about other versions) the application itself shows similar dialog in this case. Regarding on this, I doubt if this plugin's pop-up will be useful.

**Comment by AntonFisher1** [comment]
Here is a video demonstrating this existing dialog coming from the desktop application, with Yes / No options.
https://www.screencast.com/t/wMCmSUSN

**Comment by bravozulu1** [comment]
I'm having trouble playing the flash video on mac.  Let me describe a detailed use case.

Use Case:

1.  A user has an existing account saved in KeyPass

2.  They navigate to a login page and do NOT want to use existing account.  The user dismisses popup prompt  to use an existing entry and they manually enters incorrect password.

3.  A yes/no dialog should ask the user if they want to save the new password and overwrite the existing password.  Currently the good password is lost in the password database.

**Comment by bravozulu1** [comment]
This is the behavior of Dashlane.  They ask the user before replacing existing password.  See screen shot.

![sign_in_to_github_ _github](https://user-images.githubusercontent.com/10087224/41857793-f0d071e4-785d-11e8-8579-b619ddae44c6.jpg)

**Comment by AntonFisher1** [comment]
@bravozulu1 The idea is understood, but let me clarify.
As for me, this can make sense *only* if user manually enters the *existing KeePass username*, but wrong password. Because if user enters *different username* (e.g. user had abc1 / 123 credentials, and enters abc55 / 12345) , it will not overwrite anything, but just add new entry to Keepass database.
Am I right?

**Comment by bravozulu1** [comment]
Yes, this is correct.    New username works as expected.  The problem is existing username and manually entered wrong password.
